### ISSUE TITLE
How to fine-tune a U-Net pre-trained using this library?

### ISSUE THREAD

**Issue by pity2003** [jakeret/tf_unet#253]
Given I have pre-trained a model using the code, how to fine-tune it using a different dataset? How to replace the soft-max or freeze Conv layers?

Thanks a lot.

**Comment by jakeret** [comment]
The package doesn't provide an out of the box solution for this. You could use the list of[variables](https://github.com/jakeret/tf_unet/blob/master/tf_unet/unet.py#L198) and pass and adapter version of it to the [minimize function](https://github.com/jakeret/tf_unet/blob/master/tf_unet/unet.py#L347)

**Comment by pity2003** [comment]
> The package doesn't provide an out of the box solution for this. You could use the list of[variables](https://github.com/jakeret/tf_unet/blob/master/tf_unet/unet.py#L198) and pass and adapter version of it to the [minimize function](https://github.com/jakeret/tf_unet/blob/master/tf_unet/unet.py#L347)

Thank you very much for the reply. I had seen some people did fine-tuning using the method that you mentioned. But for the source code, where and how can I do it if I would like to replace the softmax? In the "_init_" of Class "unet", I can see "logits, self.variables, self.filters, self.offset = create_conv_net(self.x, self.keep_prob, channels, n_class, layers,...)". Do I need to do something for "logits"?

It would be great if you could give me a specific example for doing that.

Thanks again.

**Comment by jakeret** [comment]
https://github.com/jakeret/tf_unet/search?q=softmax&unscoped_q=softmax

**Comment by pity2003** [comment]
> https://github.com/jakeret/tf_unet/search?q=softmax&unscoped_q=softmax

Sorry - I did not follow you. Have no idea why you sent me the link of two source files.

It would be appreciated if you could provide a specific example for doing fine-tuning with you implementation.

Thanks.
